Description: Deploy html files
Author: Valerio Valerio <vdv100@gmail.com>
diff --git a/doc/src/doc.pri b/doc/src/doc.pri
index 784c881..63cdb2f 100644
--- a/doc/src/doc.pri
+++ b/doc/src/doc.pri
@@ -44,3 +44,9 @@ OTHER_FILES = $$HELP_DEP_FILES \
               $$_PRO_FILE_PWD_/doc/src/api/api-pages.qdoc \
               $$_PRO_FILE_PWD_/doc/src/api/classhierarchy.qdoc \
               $$_PRO_FILE_PWD_/doc/src/api/groups.qdoc
+
+# TEMP: deploying existing html files
+html.files = $$_PRO_FILE_PWD_/doc/html/*
+html.path = $$QMF_INSTALL_ROOT/share/doc/qmf/html
+html.CONFIG += no_check_exist
+INSTALLS += html qch_docs
